2023/README.md

```diff
@@ -0,0 +1,47 @@
+# Interop 2023
+
+Interop 2023 aims to make the web more interoperable in key areas, prioritized by web developer and user needs. The team is formed of browser vendors and other contributors to the web platform, and is part of [the web-platform-tests project](https://github.com/web-platform-tests/wpt).
+
+Similar to [Interop 2022](https://wpt.fyi/interop-2022), this will be a public metric based on the progress of selected focus areas and investigation efforts. The overall timeline for the planning process is:
+
+- Public call for proposals beginning September 15.
+- Proposal review by the team, with an opportunity for refinement.
+- Selection of proposals by the team, based on consensus.
+- Public launch in early 2023.
+
+If you would like to make a proposal or contribute in other ways, read on.
+
+## Making a proposal
+
+If you've had problems using a feature on the web because of differences between browsers, or because it isn't implemented in all browsers, it may be a good proposal for Interop 2023!
+
+Before making a proposal, here's what to expect:
+
+- Only features which have high quality specifications and tests are in scope. Interop 2023 is not a venue for specifying new features, that work happens in working groups within organizations such as W3C and WHATWG.
+- Interop 2023 is not a process for making browser vendors work on things they're opposed to. Decisions are made by consensus, so highly contentious features are unlikely to be accepted.
+- Even great proposals may ultimately not be accepted, since we have to prioritize.
```

Comment:
I'm not sure this bullet is adding much value, but it does feel like it's adding a lot of stop energy. Maybe it would sound less negative to fold it into the previous bullet point and added some positive advice: ", so highly contentious features are unlikely to be accepted, and vendors will also consider the priority of proposals against other possible work. Proposals accompanied by strong rationale (e.g. evidence of problems affecting real sites, or significant developer interest) are more likely to succeed."

Reply:
I'm fine with whatever wording others are happy with here. The purpose of this was to avoid disappointment if we end up not accepting some good proposals from "external" contributors who put in a lot of work.